images/linux/scripts/installers/mysql.sh

```diff
@@ -9,19 +9,20 @@ source $HELPER_SCRIPTS/document.sh
 
 export ACCEPT_EULA=Y
 
-# Install MySQL Client
-apt-get install mysql-client -y
-
-# InstallMySQL database development files
-apt-get install libmysqlclient-dev -y
-
 # Install MySQL Server
 MYSQL_ROOT_PASSWORD=root
 echo "mysql-server mysql-server/root_password password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
 echo "mysql-server mysql-server/root_password_again password $MYSQL_ROOT_PASSWORD" | debconf-set-selections
-
+debconf-set-selections <<< 'mysql-apt-config mysql-apt-config/select-server select mysql-8.0'
+package_version=$(curl https://dev.mysql.com/downloads/repo/apt/ 2> /dev/null | grep ".deb" | awk -F"[()]" '{print $2}' | grep "\.deb")
```

Comment:
Why do we need two greps?
`curl https://dev.mysql.com/downloads/repo/apt/ 2> /dev/null | grep "\.deb" | awk -F"[()]" '{print $2}'`

Reply:
thank you. resolved